src/libsyntax/attr.rs

```diff
@@ -877,10 +886,38 @@ pub fn find_repr_attrs(diagnostic: &Handler, attr: &Attribute) -> Vec<ReprAttr>
                     if let Some(h) = hint {
                         acc.push(h);
                     }
+                } else if let Some((name, value)) = item.name_value_str() {
```

Comment:
Sure, `[repr(align(16))]` is much nicer. Do you know what else is using that form so I can see how it's implemented?